### PR TITLE
[ko] fix: All hidden about Browser Compatibility can be deleted

### PR DESCRIPTION
**Pull request by hochan222**
issue: #1008 

세부 내용: https://github.com/mdn/content/issues/2228 참고

- [x] 추가로 발견하지 못한 항목에 대해 이후 삭제 가이드를 작성할 예정입니다. 

사용된 스크립트는 https://gist.github.com/hochan222/ad801f7a6325e2b2e391ab634d6d18ce 와 같습니다.

- 정규표현식 (추가)
```
<.*? class="hidden">.*?<a href="https:\/\/github.com\/mdn\/browser-compat-data">https:\/\/github.com\/mdn\/browser-compat-data<\/a>.*?<\/.*?>
```

```
<.*? class="hidden">.*?<.*?>.*?<a href="https:\/\/github.com\/mdn\/browser-compat-data">https:\/\/github.com\/mdn\/browser-compat-data<\/a>.*?<\/.*?>.*?<\/.*?>
```

```
<.*? class="hidden">.*?<a href="https:\/\/github.com\/mdn\/browser-compat-data">https:\/\/github.com\/mdn\/browser-compat-data<\/a>.*?<\/.*?>
```

```
<.*? class="hidden">\s<.*?>.*?<a href="https:\/\/github.com\/mdn\/browser-compat-data">https:\/\/github.com\/mdn\/browser-compat-data<\/a>.*?<\/.*?>\s<\/.*?>
```

### 기본 항목

- 1
```html
<div class="hidden">The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```

- 2
```html
<div class="hidden">
<p>The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</p>
</div>
```

- 3
```html
<p class="hidden">The compatibility table in this page is generated from structured data.
  If you'd like to contribute to the data, please check out <a
    href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>
  and send us a pull request.</p>
```

### 한국 locale 추가 항목

- 1
```html
<div class="hidden">이 설명자는 아직 <a href="https://github.com/mdn/data/blob/master/css/at-rules.json">https://github.com/mdn/data/blob/master/css/at-rules.json</a>에 등재되지 않았습니다.</div>
```
- 2
```html
<div class="hidden">
<p>The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</p>
</div>
```
- 3
```html
<div class="hidden">To contribute to this compatibility data, please write a pull request against this repository: <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>.</div>
```
- 4
```html
<div class="hidden">이 페이지와 호환되는 표가 구조화된 데이터에서 생성되게 됩니다. 당신이 이 데이터에 이바지하기를 원한다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 를검토해보시고, 우리에게 여러가지 요청을 보내보세요.</div>
```
- 5
```html
div class="hidden">이 페이지의 호환성 표는 구조화된 데이터에서 생성됩니다. 만약 데이터에 기여하고 싶다면,   <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>을 확인하여 pull request를 보내주세요.</div>
```
- 6
```html
<div class="hidden">이 페이지의 호환성 표는 구조화된 데이터에서 생성됩니다. 만약 데이터에 기여하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 을 확인하여 pull request를 보내주세요</div>
```
- 7
```html
<div class="hidden">The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```
- 8
```html
<div class="hidden">이 페이지에 호환성 표는 구조화된 데이터에서 생성됩니다. 만약 이 데이터에 공헌하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>을 확인한 다음에 요청을 보내주십시오.</div>
```
- 9
```html
<div class="hidden">
<p>The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</p>
</div>
```
- 10
```html
<div class="hidden"><font><font>이 페이지의 호환성 표는 구조화 된 데이터에서 생성됩니다. </font><font>데이터에 기여하고 싶다면 </font></font><a href="https://github.com/mdn/browser-compat-data"><font><font>https://github.com/mdn/browser-compat-data</font></font></a><font><font> 에서 풀 리퀘스트를 보내주세요.</font></font></div>
```
- 11
```html
<div class="hidden">
<p>The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</p>
</div>
```
- 12
```html
<p>The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</p>
```
- 13
```html
<p class="hidden">The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</p>
```
- 14
```html
<div class="hidden">The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a class="external" href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```
- 15
```html
<div class="hidden">The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a class="external" href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```
- 16
```html
<div class="hidden">
<p>The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>and send us a pull request.</p>
</div>
```
- 17
```html
<div class="hidden">The compatibility table in this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```
- 18
```html
<div class="hidden">The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
```
- 19
```html
<div class="hidden">
<p><span style="">The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out </span><a href="https://github.com/mdn/browser-compat-data" style="">https://github.com/mdn/browser-compat-data</a><span style=""> and send us a pull request.</span></p>
</div>
```
- 20
```html
<p class="hidden">이 페이지의 호환성표는 구조화된 데이터로부터 생성되었습니다. 만약 그 데이터에 기여하고 싶으시면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 를 참고하시고 요청을 보내주시기 바랍니다.</p>
```
- 21
```html
<div class="hidden">이 페이지의 호환성 표는 구조화된 데이터에서 생성됩니다. 만약 데이터의 기여하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>를 확인하고 pull 요청을 우리에게 보내세요.</div>
```
- 22
```html
<div class="hidden">이 페이지의 호환성 표는 구조화된 데이터에서 생성됩니다. 만약 데이터의 기여하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>를 확인하고 pull 요청을 우리에게 보내세요.</div>
```
- 23
```html
<div class="hidden">이 페이지의 호환성 표는 구조화된 데이터로부터 생성되었습니다. 데이터에 기여하시려면, <a class="external" href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 를 확인한 뒤 저희에게 풀 리퀘스트를 보내주세요.</div>
```
- 24
```html
<div class="hidden">이 페이지의 호환성 표는 구조화된 데이터에서 생성된다. 만약 당신이 데이터에 기여하고 싶다면, https://github.com/mdn/browser-compat-data을 확인하여 pull request 를 보내주십시오.<br>
```
- 25
```html
<div class="hidden">이 호환성 표는 구조화된 데이터에서 자동 생성되었습니다. 이 데이터에 참여를 원하시면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 으로 이동하셔서 pull request를 하시면 됩니다.</div>
```
- 26
```html
<p class="hidden">현재 페이지에 있는 호환성 표는 구조화된 데이터가 생성합니다. 데이터에 기여하고 싶다면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>를 확인하고 pull request를 보내세요.</p>
```
- 27
```html
<p class="hidden">이 페이지의 호환성 표는 구조화된 데이터로부터 생성되었습니다. 만약 당신이 그 데이터에 기여하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 를 체크아웃하고 풀리퀘스트를 보내주십시오.</p>
```

### CODE REVIEW

**Comment by hochan222** [comment]
I uploaded the python script by mistake and deleted it.

File: files/ko/web/api/storage/key/index.html [hunk 1]
```diff
@@ -63,11 +63,7 @@ <h2 id="Specifications">Specifications</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<div class="hidden">이 페이지의 호환성 테이블은 구조화 된 데이터에서 생성됩니다. 데이터에 기여하고 싶다면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 를 확인하고 Pull Request 를 보내주세요.</div>
-
-<p>{{Compat("api.Storage.key")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("api.Storage.key")}}</p>
```
Comment (hochan222) [inline files/ko/web/api/storage/key/index.html:66]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("api.Storage.key")}}</p>
```

File: files/ko/web/api/angle_instanced_arrays/index.html [hunk 2]
```diff
@@ -69,7 +69,7 @@ <h2 id="사양"><font><font>사양</font></font></h2>
 
 <h2 id="브라우저_호환성"><font><font>브라우저 호환성</font></font></h2>
 
-<p class="hidden"><font><font>이 페이지의 호환성 테이블은 구조화 된 데이터에서 생성됩니다. </font><font>데이터에 기여하고 싶다면 </font></font><a href="https://github.com/mdn/browser-compat-data"><font><font>https://github.com/mdn/browser-compat-data</font></font></a><font><font> 를 확인하고 </font><font>당기 요청을 보내주십시오.</font></font></p>
+<font>당기 요청을 보내주십시오.</font></font></p>
```
Comment (hochan222) [inline files/ko/web/api/angle_instanced_arrays/index.html:72]:
```suggestion
```

File: files/ko/web/api/batterymanager/dischargingtime/index.html [hunk 3]
```diff
@@ -57,11 +57,7 @@ <h2 id="설명서">설명서</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<div class="hidden">이 페이지와 호화되는 표는 구조화된 데이터에서 생성됩니다. 당신이 데이터에 기여하기를 원한다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 를 검토하시고서 우리에게 여러모로 요청을 보내보세요.</div>
-
-<p>{{Compat("api.BatteryManager.dischargingTime")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("api.BatteryManager.dischargingTime")}}</p>
```
Comment (hochan222) [inline files/ko/web/api/batterymanager/dischargingtime/index.html:60]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("api.BatteryManager.dischargingTime")}}</p>
```

File: files/ko/web/api/document/drag_event/index.html [hunk 4]
```diff
@@ -153,11 +153,7 @@ <h2 id="사양">사양</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<div class="hidden">이 호환성 테이블은 구조화된 데이터로부터 생성됩니다. 데이터에 기여하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>를 확인하여 pull request를 보내주세요.</div>
-
-<p>{{Compat("api.Document.drag_event")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("api.Document.drag_event")}}</p>
```
Comment (hochan222) [inline files/ko/web/api/document/drag_event/index.html:156]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("api.Document.drag_event")}}</p>
```

File: files/ko/web/html/global_attributes/class/index.html [hunk 5]
```diff
@@ -50,11 +50,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<div class="hidden">이 페이지에 있는 호환성 테이블은 구조적 데이터로부터 생성된 것입니다. 이 데이터에 기여하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>를 살펴보시고 pull request를 보내주시기 바랍니다.</div>
-
-<p>{{Compat("html.global_attributes.class")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("html.global_attributes.class")}}</p>
```
Comment (hochan222) [inline files/ko/web/html/global_attributes/class/index.html:53]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("html.global_attributes.class")}}</p>
```

File: files/ko/web/html/global_attributes/itemscope/index.html [hunk 6]
```diff
@@ -263,11 +263,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<div class="hidden">이 페이지의 호환성 테이블은 구조화된 데이터에서 만려면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 을 방문하셔서 요청해주시길 바랍니다.</div>
-
-<p>{{Compat("html.global_attributes.itemscope")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("html.global_attributes.itemscope")}}</p>
```
Comment (hochan222) [inline files/ko/web/html/global_attributes/itemscope/index.html:266]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("html.global_attributes.itemscope")}}</p>
```

File: files/ko/web/html/global_attributes/tabindex/index.html [hunk 7]
```diff
@@ -78,11 +78,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<div class="hidden">The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</div>
-
-<p>{{Compat("html.global_attributes.tabindex")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("html.global_attributes.tabindex")}}</p>
```
Comment (hochan222) [inline files/ko/web/html/global_attributes/tabindex/index.html:81]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("html.global_attributes.tabindex")}}</p>
```

File: files/ko/web/http/headers/accept-charset/index.html [hunk 8]
```diff
@@ -65,11 +65,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">이 페이지의 호환성 테이블은 구조화된 데이터로부터 생성되었습니다. 이 데이터에 기여하고자 한다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>을 확인하고 pull request를 보내주세요.</p>
-
-<p>{{Compat("http.headers.Accept-Charset")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("http.headers.Accept-Charset")}}</p>
```
Comment (hochan222) [inline files/ko/web/http/headers/accept-charset/index.html:68]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("http.headers.Accept-Charset")}}</p>
```

File: files/ko/web/http/headers/accept-encoding/index.html [hunk 9]
```diff
@@ -94,11 +94,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">이 페이지의 호환성 테이블은 구조화 데이터로부터 생성된 것입니다. 이 데이터에 기여하고자 한다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>을 확인하고 pull request를 보내주시기 바랍니다.</p>
-
-<p>{{Compat("http.headers.Accept-Encoding")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("http.headers.Accept-Encoding")}}</p>
```
Comment (hochan222) [inline files/ko/web/http/headers/accept-encoding/index.html:97]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("http.headers.Accept-Encoding")}}</p>
```

File: files/ko/web/http/headers/content-security-policy/index.html [hunk 10]
```diff
@@ -242,11 +242,7 @@ <h2 id="Specifications">Specifications</h2>
  </tbody>
 </table>
 
-<h2 id="Browser_compatibility">Browser compatibility</h2>
-
-<p class="hidden">The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</p>
-
-<p>{{Compat("http.headers.csp.Content-Security-Policy")}}</p>
+<h2 id="Browser_compatibility">Browser compatibility<p>{{Compat("http.headers.csp.Content-Security-Policy")}}</p>
```
Comment (hochan222) [inline files/ko/web/http/headers/content-security-policy/index.html:245]:
```suggestion
<h2 id="Browser_compatibility">Browser compatibility</h2>

<p>{{Compat("http.headers.csp.Content-Security-Policy")}}</p>
```

File: files/ko/web/http/headers/pragma/index.html [hunk 11]
```diff
@@ -72,7 +72,7 @@ <h2 id="브라우저_호환성">브라우저 호환성</h2>
 
 <p class="hidden">이 페이지의 호환성 테이블은 특정하게 구조화된 데이터를 기반으로 만들어졌다. 당신이 만일 이 데이터에 기여하고 싶다면 아래의 링크를 따라가 우리에게 Pull Request 를 보내면 된다.</p>
 
-<p class="hidden"><a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a></p>
+
```
Comment (hochan222) [inline files/ko/web/http/headers/pragma/index.html:75]:
```suggestion
```

File: files/ko/web/http/status/301/index.html [hunk 12]
```diff
@@ -40,11 +40,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">이 페이지의 호환성 표는 구조적인 데이터로부터 생성되었습니다. 데이터에 기여하고자 하시면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>를 참조하셔서 pull request를 보내주시기 바랍니다.</p>
-
-<p>{{Compat("http.status.301")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("http.status.301")}}</p>
```
Comment (hochan222) [inline files/ko/web/http/status/301/index.html:43]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("http.status.301")}}</p>
```

File: files/ko/web/http/status/403/index.html [hunk 13]
```diff
@@ -36,11 +36,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">이 페이지의 호환성 표는 구조화된 데이터에서 생성됩니다. 만약 이 데이터에 기여하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 를 방문하고 full-request를 보내주세요.</p>
-
-<p>{{Compat("http/status", "403")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("http/status", "403")}}</p>
```
Comment (hochan222) [inline files/ko/web/http/status/403/index.html:39]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("http/status", "403")}}</p>
```

File: files/ko/web/javascript/reference/functions/get/index.html [hunk 14]
```diff
@@ -159,11 +159,7 @@ <h2 id="스펙">스펙</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">The compatibility table on this page is generated from structured data. If you'd like to contribute to the data, please check out <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> and send us a pull request.</p>
-
-<p>{{Compat("javascript.functions.get")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("javascript.functions.get")}}</p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/functions/get/index.html:162]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("javascript.functions.get")}}</p>
```

File: files/ko/web/javascript/reference/global_objects/error/name/index.html [hunk 15]
```diff
@@ -58,7 +58,7 @@ <h2 id="브라우저_호환성">브라우저 호환성</h2>
 
 <div>
 <div>
-<div class="hidden">이 페이지의 호환성 테이블은 구조화된 데이터로부터 생성됩니다. 해당 데이터를 개선하고 싶다면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>을 체크아웃하고 저희에게 풀 리퀘스트를 보내주십시오.</div>
+
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/error/name/index.html:61]:
```suggestion
```

File: files/ko/web/javascript/reference/global_objects/function/arguments/index.html [hunk 16]
```diff
@@ -47,7 +47,7 @@ <h2 id="Specifications">Specifications</h2>
 <h2 id="Browser_호환성">Browser 호환성</h2>
 
 <div>
-<div class="hidden">이 페이지에 있는 호환성 테이블은 구조화된 데이터로 생성 되었습니다. 이 데이터에 기여를 하기 원하시면, 이 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 페이지를 확인 하시고 Pull Request를 보내주시기 바랍니다.</div>
+
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/function/arguments/index.html:50]:
```suggestion
```

File: files/ko/web/javascript/reference/global_objects/math/sign/index.html [hunk 17]
```diff
@@ -95,11 +95,7 @@ <h2 id="규격명세">규격명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">이 페이지에서 호환성 표는 구조화된 자료로부터 생성됩니다. 그 자료에 기여하시려면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 를 확인하시고 저희에게 pull 요청을 보내십시오.</p>
-
-<p>{{Compat("javascript.builtins.Math.sign")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("javascript.builtins.Math.sign")}}</p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/math/sign/index.html:98]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("javascript.builtins.Math.sign")}}</p>
```

File: files/ko/web/javascript/reference/global_objects/math/sqrt1_2/index.html [hunk 18]
```diff
@@ -45,11 +45,7 @@ <h2 id="사양">사양</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden"><span>이 페이지의 호환성 표는 구조화된 데이터에서 생성됩니다.</span> 데이터에 기여하고 싶으시다면,  <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>을 확인하고 풀 리퀘스트를 보내 주시기 바랍니다.</p>
-
-<p>{{Compat("javascript.builtins.Math.SQRT1_2")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("javascript.builtins.Math.SQRT1_2")}}</p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/math/sqrt1_2/index.html:48]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("javascript.builtins.Math.SQRT1_2")}}</p>
```

File: files/ko/web/javascript/reference/global_objects/promise/finally/index.html [hunk 19]
```diff
@@ -85,11 +85,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">To contribute to this compatibility data, please write a pull request against this repository: <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>.</p>
-
-<p>{{Compat("javascript.builtins.Promise.finally")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("javascript.builtins.Promise.finally")}}</p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/promise/finally/index.html:88]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("javascript.builtins.Promise.finally")}}</p>
```

File: files/ko/web/javascript/reference/global_objects/promise/resolve/index.html [hunk 20]
```diff
@@ -149,9 +149,7 @@ <h2 id="명세">명세</h2>
 
 <h2 id="브라우저_호환성">브라우저 호환성</h2>
 
-<div class="hidden">
-<p>To contribute to this compatibility data, please write a pull request against this repository: <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>.</p>
-</div>
+
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/promise/resolve/index.html:152]:
```suggestion
```

File: files/ko/web/javascript/reference/global_objects/referenceerror/index.html [hunk 21]
```diff
@@ -116,7 +116,7 @@ <h2 id="명세">명세</h2>
 <h2 id="브라우저_호환성">브라우저 호환성</h2>
 
 <div>
-<div class="hidden">이 페이지의 호환성 테이블은 구조화된 데이터로부터 생성됩니다. 해당 데이터를 개선하고 싶다면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>을 체크아웃하고 저희에게 풀 리퀘스트를 보내주십시오.</div>
+
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/referenceerror/index.html:119]:
```suggestion
```

File: files/ko/web/javascript/reference/global_objects/set/add/index.html [hunk 22]
```diff
@@ -68,11 +68,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<div class="hidden">본 페이지의 호환성 표는 구조화된 데이터로부터 생성되었습니다. 만약 당신이 데이터에 기여를 원한다면 <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>를 체크아웃하고 우리에게 pull request를 보내주세요.</div>
-
-<p>{{Compat("javascript.builtins.Set.add")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("javascript.builtins.Set.add")}}</p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/set/add/index.html:71]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("javascript.builtins.Set.add")}}</p>
```

File: files/ko/web/javascript/reference/global_objects/string/match/index.html [hunk 23]
```diff
@@ -128,11 +128,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">이 페이지의 호환성 표는 구조화된 데이터를 기반으로 작성하였습니다. 데이터를 제공하고 싶으시다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a>를 체크하시고, 풀 리퀘스트를 보내주시기 바랍니다.</p>
-
-<p>{{Compat("javascript.builtins.String.match")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("javascript.builtins.String.match")}}</p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/string/match/index.html:131]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("javascript.builtins.String.match")}}</p>
```

File: files/ko/web/javascript/reference/global_objects/string/tostring/index.html [hunk 24]
```diff
@@ -47,7 +47,7 @@ <h2 id="명세서"><font><font>명세서</font></font></h2>
 
 <h2 id="브라우저_호환성"><font><font>브라우저 호환성</font></font></h2>
 
-<p class="hidden"><font><font>이 페이지의 호환성 표는 구조화 된 데이터에서 생성됩니다. </font><font>데이터에 기여하려면 </font></font><a href="https://github.com/mdn/browser-compat-data"><font><font>https://github.com/mdn/browser-compat-data</font></font></a><font><font> 를 확인하고 </font><font>풀 요청을 보내주십시오.</font></font></p>
+<font>풀 요청을 보내주십시오.</font></font></p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/string/tostring/index.html:50]:
```suggestion
```

File: files/ko/web/javascript/reference/global_objects/string/valueof/index.html [hunk 25]
```diff
@@ -69,11 +69,7 @@ <h2 id="명세">명세</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<p class="hidden">이 페이지의 호환성 표는 구조화된 데이터로부터 생성됩니다. 만약 데이터에 기여하고 싶다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 을 확인하시고 pull 요청을 보내주세요.</p>
-
-<p>{{Compat("javascript.builtins.String.valueOf")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("javascript.builtins.String.valueOf")}}</p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/global_objects/string/valueof/index.html:72]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("javascript.builtins.String.valueOf")}}</p>
```

File: files/ko/web/javascript/reference/statements/do...while/index.html [hunk 26]
```diff
@@ -64,11 +64,7 @@ <h2 id="자세한_내용">자세한 내용</h2>
  </tbody>
 </table>
 
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
-
-<div class="hidden">이 페이지의 브라우저 호환성 표는 구조화된 데이터에서 생성되었습니다. 만약 이 문서에 기여하고 싶으시다면, <a href="https://github.com/mdn/browser-compat-data">https://github.com/mdn/browser-compat-data</a> 문서를 확인하고 수정한 뒤 pull request 를 보내주십시오.</div>
-
-<p>{{Compat("javascript.statements.do_while")}}</p>
+<h2 id="브라우저_호환성">브라우저 호환성<p>{{Compat("javascript.statements.do_while")}}</p>
```
Comment (hochan222) [inline files/ko/web/javascript/reference/statements/do...while/index.html:67]:
```suggestion
<h2 id="브라우저_호환성">브라우저 호환성</h2>

<p>{{Compat("javascript.statements.do_while")}}</p>
```

**Review by hochan222 — COMMENTED** [review]
검토 완료.

**Review by peterbe — APPROVED** [review]

**Comment by peterbe** [comment]
Impressive work @hochan222 
Can you merge in `main` to resolve the conflict then we can squash and merge this.

**Comment by hochan222** [comment]
@peterbe 

Thank you. I just dealt with it and we'll merge it. @GwangYeol-Im

**Comment by peterbe** [comment]
I can merge it now if you want?

**Comment by hochan222** [comment]
Sorry. I just got a message from gim saying that it will merge and I was waiting, but I'll merge it!

**Pull request merged**